Gemfile

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 source 'https://rubygems.org'
+ruby '2.4.1'
```

Comment:
@travelist @victorfonsec4 @kosukemori 
I added the ruby version in Gemfile. Do you have any concerns about it?
The reason that I did this is hoping to let other developers and users know what version of ruby we are using while implementing it. But, actually, the client will work with other version of ruby as well...